Tests/SQLiteNIOTests/SQLiteCustomFunctionTests.swift

```diff
@@ -341,21 +341,25 @@ class DatabaseFunctionTests: XCTestCase {
 	func testFunctionsAreClosures() throws {
 		let conn = try SQLiteConnection.open(storage: .memory, threadPool: self.threadPool, on: self.eventLoop).wait()
 		defer { try! conn.close().wait() }
-
-		var x = 123
+        
+        final class UnsafeMutableTransfer<T>: @unchecked Sendable {
```

Comment:
You could just copy-paste the same thing which is also named `UnsafeMutableTransferBox`, not `UnsafeMutableTransfer`(being extra nit-picky today 😛)

Reply:
For some reason GitHub is not letting me reply directly to the comment on `UnsafeMutableTransferBox` in `SQLiteConnection.swift`, so the replies to both comments are here...

- (re: "`Wrapped` should be `Sendable`") You do of course understand that the _SPECIFIC_ point of `UnsafeMutableTransferBox` is to carry non-`Sendable` values, right? This is copied from NIO's version (slightly condensed).
- (re: duplicate unsafe mutable transfer in test) It is not at all advisable to make the scope of these unsafe types any greater than it needs to be, their being unsafe any all. As such it's far preferable to let the test have its own version than to rely on snagging the one from the main code. As for the naming, gosh, how will I ever live with myself? 🤣

Reply:
> You do of course understand that the SPECIFIC point of UnsafeMutableTransferBox is to carry non-Sendable values, right?

My brain is a little bit foggy thanks to covid but i think not?
The point (here) is to make a _value type_, _mutable_. Be it Sendable or non-Sendable. You need to use `var` which the compiler doesn't like for mutating anything (in the context of what you're trying to do). If the value type is Sendable then mutating it could be less unsafe, so you could add the `Sendable` constraint.

Reply:
> My brain is a little bit foggy thanks to covid but i think not? The point (here) is to make a _value type_, _mutable_. Be it Sendable or non-Sendable. You need to use `var` which the compiler doesn't like for mutating anything (in the context of what you're trying to do). If the value type is Sendable then mutating it could be less unsafe, so you could add the `Sendable` constraint.

If it's being mutated, it doesn't matter whether it's `Sendable` or not, it's unsafe either way, and requiring that it be `Sendable` doesn't have any practical value. But, there you go, I added the constraint while cleaning up, because I've heard less sensible concerns and why ignore yours just because I'm unreasonable no matter how hard you try to make sense? 🙃 (Note: I am, in fact, unreasonable and you are, in fact, making sense - that's my point xD).